### ISSUE TITLE
[Windows] TypeError: unable to resolve type 'uintptr_t'

### ISSUE THREAD

**Issue by SubhoBoy** [ffi/ffi#939]
Running Windows 11 21H2,
```
$ ruby.exe -v
ruby 2.7.5p203 (2021-11-24 revision f69aeb8314) [x64-mingw32]
```
Full trace:-
```
$  bundle exec rake secret --trace
rake aborted!
TypeError: unable to resolve type 'uintptr_t'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/ffi-1.12.2-x64-mingw32/lib/ffi/types.rb:69:in `find_type'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/ffi-1.12.2-x64-mingw32/lib/ffi/library.rb:444:in `typedef'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rest-client-2.0.2-x64-mingw32/lib/restclient/windows/root_certs.rb:17:in `<class:RootCerts>'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rest-client-2.0.2-x64-mingw32/lib/restclient/windows/root_certs.rb:12:in `<top (required)>'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rest-client-2.0.2-x64-mingw32/lib/restclient/windows.rb:7:in `require_relative'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rest-client-2.0.2-x64-mingw32/lib/restclient/windows.rb:7:in `<top (required)>'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `block in require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:291:in `load_dependency'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rest-client-2.0.2-x64-mingw32/lib/restclient.rb:18:in `<top (required)>'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `block in require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:291:in `load_dependency'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rturk-2.12.1/lib/rturk/requester.rb:7:in `<top (required)>'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `block in require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:291:in `load_dependency'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/activesupport-6.0.4.4/lib/active_support/dependencies.rb:324:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rturk-2.12.1/lib/rturk.rb:30:in `<top (required)>'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/bundler-2.3.5/lib/bundler/runtime.rb:60:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/bundler-2.3.5/lib/bundler/runtime.rb:60:in `block (2 levels) in require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/bundler-2.3.5/lib/bundler/runtime.rb:55:in `each'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/bundler-2.3.5/lib/bundler/runtime.rb:55:in `block in require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/bundler-2.3.5/lib/bundler/runtime.rb:44:in `each'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/bundler-2.3.5/lib/bundler/runtime.rb:44:in `require'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/bundler-2.3.5/lib/bundler.rb:176:in `require'
D:/Programs/huginn/config/application.rb:7:in `<top (required)>'
D:/Programs/huginn/Rakefile:5:in `require_relative'
D:/Programs/huginn/Rakefile:5:in `<top (required)>'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/rake_module.rb:29:in `load'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/rake_module.rb:29:in `load_rakefile'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/application.rb:710:in `raw_load_rakefile'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/application.rb:104:in `block in load_rakefile'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/application.rb:186:in `standard_exception_handling'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/application.rb:103:in `load_rakefile'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/application.rb:82:in `block in run'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/application.rb:186:in `standard_exception_handling'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/lib/rake/application.rb:80:in `run'
C:/ProgramData/Ruby/Ruby27-x64/lib/ruby/gems/2.7.0/gems/rake-13.0.6/exe/rake:27:in `<top (required)>'
C:/ProgramData/Ruby/Ruby27-x64/bin/rake:25:in `load'
C:/ProgramData/Ruby/Ruby27-x64/bin/rake:25:in `<main>'
```
I came across #386 and added `rbx.platform.typedef.uintptr_t = uint64` to `C:\ProgramData\Ruby\Ruby27-x64\lib\ruby\gems\2.7.0\gems\ffi-1.12.2-x64-mingw32\lib\ffi\platform\x86_64-windows\types.conf`, however this seems to have no effect.

**Comment by larskanis** [comment]
This has been fixed in ffi-1.13.0. Please upgrade!

**Comment by SubhoBoy** [comment]
Yes, it works now! Thanks and sorry for the inconvenience!
